Source/RN4UE4/RakNetRP.cpp

```diff
@@ -4,6 +4,7 @@
 #include <functional>
 #include <string>
 #include "ReplicaBase.h"
+#include "Engine.h"
```

Comment:
Why is this needed?

Reply:
It gives an error with GEngine because I didn't recognize it class

Reply:
What is new that needs it?